x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/service_group_card.tsx

```diff
@@ -73,9 +76,27 @@ export function ServiceGroupsCard({
     href,
   };
 
-  return (
-    <EuiFlexItem key={serviceGroup.groupName}>
-      <EuiCard layout="vertical" {...cardProps} />
-    </EuiFlexItem>
+  const Card = <EuiCard layout="vertical" {...cardProps} />;
+  const component = withTour ? (
+    <ServiceGroupsTour
+      type="serviceGroupCard"
+      title={i18n.translate(
+        'xpack.apm.serviceGroups.tour.serviceGroups.title',
+        { defaultMessage: 'All services group' }
+      )}
+      content={i18n.translate(
+        'xpack.apm.serviceGroups.tour.serviceGroups.content',
+        {
+          defaultMessage:
+            "Now that you've created a service group, your All services inventory has moved here. This group cannot be edited or removed.",
```

Comment:
The wording here sounds a little off:
>... your All services inventory has moved here.

perhaps quotes might help?:

`... your "All services" inventory has moved here.`

I'm not sure if that's much better tho.

Reply:
Thoughts @formgeist / @boriskirov ?